cypress/e2e/users/loans/loan-reached-maximum-renewals.cy.js

```diff
@@ -215,15 +211,19 @@ describe('ui-users-loans: renewal failure because loan has reached maximum renew
     })).then(() => {
       InventoryInstance.deleteInstanceViaApi(secondInstanceIds.instanceId);
     });
-
-    cy.deleteLoanPolicy(Cypress.env(CY_ENV.LOAN_POLICY).id);
-    RequestPolicy.deleteViaApi(Cypress.env(CY_ENV.REQUEST_POLICY).id);
-    LostItemFeePolicy.deleteViaApi(Cypress.env(CY_ENV.LOST_ITEM_FEES_POLICY).id);
-    OverdueFinePolicy.deleteViaApi(Cypress.env(CY_ENV.OVERDUE_FINE_POLICY).id);
-    NoticePolicy.deleteViaApi(Cypress.env(CY_ENV.NOTICE_POLICY).id);
-
+    cy.deleteLoanPolicy(loanPolicy.id);
```

Comment:
please delete the loan type that you create in precondition

Reply:
Done